tests/components/weenect/test_init.py

```diff
@@ -0,0 +1,45 @@
+"""Test weenect setup process."""
+import pytest
+
+from homeassistant.components.weenect import (
+    WeenectDataUpdateCoordinator,
+    async_reload_entry,
+    async_setup_entry,
+    async_unload_entry,
+)
+from homeassistant.components.weenect.const import DOMAIN
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import MOCK_CONFIG
+
+from tests.common import MockConfigEntry
+
+
+@pytest.mark.usefixtures("bypass_get_trackers")
+async def test_setup_unload_and_reload_entry(hass):
+    """Test entry setup and unload."""
```

Comment:
Docstring seems to disagree with the name of the test.

```suggestion
async def test_setup_unload_and_reload_entry(hass):
    """Test entry setup, reload, and unload."""
```